microsoftgraph/microsoft-graph-explorer-v4
# Task: Persist Query Response selection

## Overview
Closes #1827 

### Demo

Optional. Screenshots, `curl` examples, etc.

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* How to test this PR
Check out this branch and run it locally
Click on any response tab
Click on the Expand button
Notice that the tab you selected first is shown on the expanded modal
Select a different tab on the modal like Code Snippets
Close the modal
Notice that the selection stays on the tab you selected
